compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToROCDL.cpp

```diff
@@ -36,14 +40,32 @@ namespace iree_compiler {
 
 namespace {
 
+// Transform gpu.barrier -> amdgpu.lds_barrier
+struct RewriteBarriers  : public OpRewritePattern<gpu::BarrierOp> {
```

Comment:
nit: Use a more descriptive pattern name, e.g. `ReplaceGPUBarrierWithLDSBarrier`

Reply:
Agree with that.